org.lflang/src/org/lflang/generator/JavaGeneratorUtils.java

```diff
@@ -222,8 +227,27 @@ public static void validateImports(
             if (
                 bad.contains(resource) || issues.size() > 0
             ) {
+                // Report the error on this resource.
+                // Need a path, which, sadly, seems impossibly difficult to get reliably.
+                // Be very paranoid here because we don't want errors to occur while error reporting.
+                // FIXME: Is this how to get a path?
```

Comment:
`fileConfig.srcFile` should do the trick: https://github.com/lf-lang/lingua-franca/blob/b7dc90e2c2df3c9dcba3eeae462ecc6edf22b6c0/org.lflang/src/org/lflang/FileConfig.java#L121)

Reply:
Thanks! That does it.